### PR TITLE
More canonical formatting for latex dmrs

### PR DESCRIPTION
**Pull request by fcbond**
Here is a new version with some of the formatting made easier to change in LaTeX, and the node ids labelled!

### CODE REVIEW

File: delphin/extra/latex.py [hunk 1]
```diff
@@ -47,59 +47,77 @@ def dmrs_tikz_dependency(xs):
...
+            return 'rstr'
+        elif link.post == EQ_POST:
+            return 'eq'
+        else:
+            return 'arg'
 
     if isinstance(xs, Xmrs):
         xs = [xs]
 
     slots = defaultdict(set)  # from/(+-)to; + for above, - for below
```
Comment (goodmami) [inline delphin/extra/latex.py]: `slots` is no longer used and can be removed

File: delphin/extra/latex.py [hunk 2]
```diff
@@ -47,59 +47,77 @@ def dmrs_tikz_dependency(xs):
...
-        lines.append(
-            '\\begin{dependency}['
-            'text only label,'
-            'label style={above,font=\\footnotesize},'
-            'edge unit distance=2ex'
-            ']'
-        )
+        lines.append("\\begin{dependency}[dmrs]")
         ns = nodes(x)
         ls = links(x)
```
Comment (goodmami) [inline delphin/extra/latex.py]: `ls` was not even used prior to the PR; but it can go too

File: delphin/extra/latex.py [hunk 3]
```diff
@@ -47,59 +47,77 @@ def dmrs_tikz_dependency(xs):
...
+                # slots[link.start].add(slot)
+                lines.append('  \\depedge[{}]{{{}}}{{{}}}{{\\srl{{{}}}}}'.format(
+                    label_edge(link),
                     nodeidx[link.start],
                     nodeidx[link.end],
                     _latex_escape(link_label(link))
                 ))
-        lines.append('\\end{dependency}')
+        ### placeholder for icons
+        lines.append('%  \\depedge[icons]{f}{t}{focus}')
```
Comment (goodmami) [inline delphin/extra/latex.py]: Shouldn't the ICONS placeholder be commented out? Currently it appears in the .tex output (although in a TeX comment).

File: delphin/extra/latex.py [hunk 4]
```diff
@@ -47,59 +47,77 @@ def dmrs_tikz_dependency(xs):
...
-                slot = link.end * -1 if side.endswith('below') else link.end
-                if slot in slots[link.start]:
-                    opts.append('edge unit distance=4ex')
-                slots[link.start].add(slot)
-                lines.append('  \\depedge[{}]{{{}}}{{{}}}{{{}}}'.format(
-                    ','.join(opts),
+                # slot = link.end * -1 if side.endswith('below') else link.end
+                # if slot in slots[link.start]:
+                #     opts.append('edge unit distance=4ex')
+                # slots[link.start].add(slot)
```
Comment (goodmami) [inline delphin/extra/latex.py]: Commented out code can just be removed

File: delphin/extra/latex.py (hunk 2 above)
Comment (fcbond) [inline delphin/extra/latex.py]: done

File: delphin/extra/latex.py (hunk 4 above)
Comment (fcbond) [inline delphin/extra/latex.py]: done

File: delphin/extra/latex.py (hunk 3 above)
Comment (fcbond) [inline delphin/extra/latex.py]: I put it there to make it easy for someone to add icons by hand in the tex if they want to.  
So it should stay in the python, commented out in the tex (where it is easy to uncomment).  
But I will capitalize FOCUS to be more consistent.

**Pull request merged**